src/patterns/ec2-app.ts

```diff
@@ -145,6 +146,7 @@ export interface GuEc2AppProps extends AppIdentity {
     [Stage.PROD]: GuAsgCapacityProps;
   };
   accessLogging?: AccessLoggingProps;
+  autoScalingGroup?: { blockDevices?: BlockDevice[] };
```

Comment:
I would probably move `blockDevices` up a level (rather than nesting it within `autoScalingGroup`), as it seems comparable to other optional props at that level (e.g. `roleConfiguration` and `accessLogging`).

Reply:
Think you're right on this, this has been changed.